docs/spec/openapi/paths/server_info.yaml

```diff
@@ -15,11 +15,3 @@ get:
   responses:
     '200':
       $ref: '../components/responses/200_server_info.yaml'
-post:
```

Comment:
[to reviewer] other than openapi spec, we'll also need to remove it from obs docs in a separate PR: https://www.elastic.co/guide/en/observability/current/apm-api-info.html

Reply:
Why would you remove it? I would add a disclaimer that the `/` endpoint is to be used fr healthcheck or test only, but I think the use case of POST with credentials (to verify credentials) is a nice one to give our user guidance on. I would definitely clarify its purpose though.

Reply:
the intention of this very PR is to remove the "functionality" to POST to `/`. As such, we are removing the corresponding part in the docs. If we don't remove it in the docs, we'll be left with docs that reference an endpoint that doesn't work.

Reply:
> I think the use case of POST with credentials (to verify credentials) is a nice one to give our user guidance on

I agree that we should update the docs to state that we can optionally provide credentials on the GET request to get additional info about the server. But I'm not sure if OpenAPI spec supports that.